sw/device/silicon_creator/lib/epmp_state.c

```diff
@@ -73,3 +76,63 @@ rom_error_t epmp_state_check(void) {
   }
   return kErrorEpmpBadCheck;
 }
+
+void epmp_print(void) {
```

Comment:
nit: s/`epmp_print`/`epmp_debug_print`/

Reply:
I've renamed this as `dbg_print_epmp()` and moved it into dbg_print.c